[2025-03-23T13:31:27.313+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: healthcare_claims_pipeline.load_csv_to_postgres manual__2025-03-23T13:26:09.691817+00:00 [queued]>
[2025-03-23T13:31:27.328+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: healthcare_claims_pipeline.load_csv_to_postgres manual__2025-03-23T13:26:09.691817+00:00 [queued]>
[2025-03-23T13:31:27.329+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-23T13:31:27.350+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_csv_to_postgres> on 2025-03-23 13:26:09.691817+00:00
[2025-03-23T13:31:27.355+0000] {standard_task_runner.py:57} INFO - Started process 551 to run task
[2025-03-23T13:31:27.358+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'healthcare_claims_pipeline', 'load_csv_to_postgres', 'manual__2025-03-23T13:26:09.691817+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/healthcare_claims_dag.py', '--cfg-path', '/tmp/tmp0w4k0cbk']
[2025-03-23T13:31:27.360+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask load_csv_to_postgres
[2025-03-23T13:31:27.408+0000] {task_command.py:415} INFO - Running <TaskInstance: healthcare_claims_pipeline.load_csv_to_postgres manual__2025-03-23T13:26:09.691817+00:00 [running]> on host f14adedb7c2a
[2025-03-23T13:31:27.497+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='healthcare_claims_pipeline' AIRFLOW_CTX_TASK_ID='load_csv_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-03-23T13:26:09.691817+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-23T13:26:09.691817+00:00'
[2025-03-23T13:31:27.512+0000] {logging_mixin.py:151} INFO - Loading CSV from path: /opt/***/data/DE1_0_2008_TO_2010_INPATIENT_CLAIMS_SAMPLE_1.csv
[2025-03-23T13:31:27.512+0000] {logging_mixin.py:151} INFO - Reading only 1000 records to save space
[2025-03-23T13:31:27.517+0000] {logging_mixin.py:151} INFO - Error loading CSV to PostgreSQL: [Errno 2] No such file or directory: '/opt/***/data/DE1_0_2008_TO_2010_INPATIENT_CLAIMS_SAMPLE_1.csv'
[2025-03-23T13:31:27.517+0000] {logging_mixin.py:151} INFO - Attempting to insert minimal sample data
[2025-03-23T13:31:27.528+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-03-23T13:31:27.553+0000] {logging_mixin.py:151} INFO - Inserted 2 sample records
[2025-03-23T13:31:27.554+0000] {python.py:194} INFO - Done. Returned value was: 2
[2025-03-23T13:31:27.581+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=healthcare_claims_pipeline, task_id=load_csv_to_postgres, execution_date=20250323T132609, start_date=20250323T133127, end_date=20250323T133127
[2025-03-23T13:31:27.610+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-23T13:31:27.650+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
